tests/test_basic.py

```diff
@@ -0,0 +1,223 @@
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_geojson_bbox_invalid_coordinates(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/muenster_ring_zeit.geojs', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid coordinates', "stderr should not be empty"
+
+def test_geojson_time(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/muenster_ring_zeit.geojs', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "2018-11-14 00:00:00" in ret.stdout,  "time value is printed to console"
+
+def test_geojson_time_invalid(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/muenster_ring_zeit.geojs', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_nc_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/ECMWF_ERA-40_subset1.nc', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_nc_time(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/ECMWF_ERA-40_subset1.nc', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "2018-11-14 00:00:00" in ret.stdout,  "time value is printed to console"
+
+def test_nc_time_invalid(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/ECMWF_ERA-40_subset1.nc', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_kml_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/aasee.kml', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
```

Comment:
This is the same bbox as the file `ECMWF_ERA-40_subset1.nc` - is that intentional? Please double-check.

Reply:
Updated from ground truth.